README.md

```diff
@@ -89,6 +89,71 @@ item.active = false
 item.save
 ```
 
+### `BatchGetItem` and `BatchWriteItem`
+Aws Record provides [BatchGetItem](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/DynamoDB/Client.html#batch_get_item-instance_method)
+and [BatchWriteItem](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/DynamoDB/Client.html#batch_write_item-instance_method)
+support for aws-record models.
+
+More info under the following documentation:
+* [Batch](https://docs.aws.amazon.com/sdk-for-ruby/aws-record/api/Aws/Record/Batch.html)
+* [BatchWrite](https://docs.aws.amazon.com/sdk-for-ruby/aws-record/api/Aws/Record/BatchWrite.html)
+* [BatchRead](https://docs.aws.amazon.com/sdk-for-ruby/aws-record/api/Aws/Record/BatchRead.html)
+
+See examples below to see the feature in action.
+
+**`BatchGetItem` Example:**
+```ruby
+class Lunch
+  include Aws::Record
+  integer_attr :id,   hash_key: true
+  string_attr  :name, range_key: true
+end
+
+class Dessert
+  include Aws::Record
+  integer_attr :id,   hash_key: true
+  string_attr  :name, range_key: true
+end
+
+# batch operations
+operation = Aws::Record::Batch.read do |db|
+  db.find(Lunch, id: 1, name: 'Papaya Salad')
+  db.find(Lunch, id: 2, name: 'BLT Sandwich')
+  db.find(Dessert, id: 1, name: 'Apple Pie')
+end
+
+# BatchRead is enumerable and handles pagination
+operation.each { |item| item.id }
+
+# unprocessed items can be processed by calling Aws::Record::BatchRead#execute!
```

Comment:
This makes it sound a little like, even after calling `.each` you may need to call `execute!`.  Maybe something like: "Alternatively, BatchRead provides a lower level interface through: execute!, complete? and items.  Unprocessed items can be processed by calling:

Reply:
Updated Comments in the example as you suggested.